src/open-data/Services.csv

```diff
@@ -1,314 +1,314 @@
-PlayFabBillingService,playfabbillingservice/egress,Azure PlayFab,Media,Microsoft,Cloud Provider
-PlayFabBillingService,playfabbillingservice/events,Azure PlayFab,Media,Microsoft,Cloud Provider
-PlayFabBillingService,playfabbillingservice/executiontimegbs,Azure PlayFab,Media,Microsoft,Cloud Provider
-PlayFabBillingService,playfabbillingservice/filestorage,Azure PlayFab,Media,Microsoft,Cloud Provider
-PlayFabBillingService,playfabbillingservice/flatfee,Azure PlayFab,Media,Microsoft,Cloud Provider
-PlayFabBillingService,playfabbillingservice/insightscredits,Azure PlayFab,Media,Microsoft,Cloud Provider
-PlayFabBillingService,playfabbillingservice/inventoryreadrequests,Azure PlayFab,Media,Microsoft,Cloud Provider
-PlayFabBillingService,playfabbillingservice/inventorywriterequests,Azure PlayFab,Media,Microsoft,Cloud Provider
-PlayFabBillingService,playfabbillingservice/itemstorage,Azure PlayFab,Media,Microsoft,Cloud Provider
-PlayFabBillingService,playfabbillingservice/lobbyrequests,Azure PlayFab,Media,Microsoft,Cloud Provider
-PlayFabBillingService,playfabbillingservice/matchmakingrequests,Azure PlayFab,Media,Microsoft,Cloud Provider
-PlayFabBillingService,playfabbillingservice/messages,Azure PlayFab,Media,Microsoft,Cloud Provider
-PlayFabBillingService,playfabbillingservice/partyconnectivity,Azure PlayFab,Media,Microsoft,Cloud Provider
-PlayFabBillingService,playfabbillingservice/playstream,Azure PlayFab,Media,Microsoft,Cloud Provider
-PlayFabBillingService,playfabbillingservice/reads,Azure PlayFab,Media,Microsoft,Cloud Provider
-PlayFabBillingService,playfabbillingservice/storage,Azure PlayFab,Media,Microsoft,Cloud Provider
-PlayFabBillingService,playfabbillingservice/telemetry,Azure PlayFab,Media,Microsoft,Cloud Provider
-PlayFabBillingService,playfabbillingservice/totalexecution,Azure PlayFab,Media,Microsoft,Cloud Provider
-PlayFabBillingService,playfabbillingservice/writes,Azure PlayFab,Media,Microsoft,Cloud Provider
+ConsumedService;ResourceType;ServiceName;ServiceCategory;PublisherName;PublisherType;Location;Service Model
+Citrix.Services;citrix.services/xenappessentials;Citrix Virtual App Essentials;Compute;Citrix;Marketplace;Cloud;SaaS
+Citrix.Services;citrix.services/xendesktopessentials;Citrix Virtual Desktop Essentials;Compute;Citrix;Marketplace;Cloud;SaaS
+GitHub.Enterprise;github.enterprise/accounts;GitHub Enterprise Cloud;Developer Tools;Microsoft;Cloud Provider;Cloud;SaaS
+GitHub;github/enterpriseaccount;GitHub Enterprise Cloud;Developer Tools;Microsoft;Cloud Provider;Cloud;SaaS
+Marketplace;marketplace/account;Azure Marketplace;Other;Microsoft;Cloud Provider;Cloud;SaaS
+MFA;mfa/perauthentication;Microsoft Entra ID;Identity;Microsoft;Cloud Provider;Cloud;PaaS
+MFA;mfa/peruser;Microsoft Entra ID;Identity;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.AAD;microsoft.aad/domainservices;Azure Active Directory Domain Services;Identity;Microsoft;Cloud Provider;Cloud;PaaS

```

Comment:
I thought identity was SaaS. We should double-check.

Reply:
IDaaS is considered as a form of SaaS, changes reflected in the table